scipy/stats/morestats.py

```diff
@@ -1696,6 +1696,21 @@ def shapiro(x):
 #             Vol. 66, Issue 3, Dec. 1979, pp 591-595.
 _Avals_logistic = array([0.426, 0.563, 0.660, 0.769, 0.906, 1.010])
 
+# From Richard A. Lockhart and Michael A. Stephens "Estimation and Tests of
+#             Fit for the Three-Parameter Weibull Distribution"
+#             Journal of the Royal Statistical Society.Series B(Methodological)
+#             Vol. 56, No. 3 (1994), pp. 491-500
+_Avals_weibull = {0: array([0.292, 0.395, 0.467, 0.522, 0.617, 0.711, 0.836, 0.931]),
```

Comment:
what are the keys of the dictionary?

Reply:
it is a 100*C-values from table 1

Reply:
thanks, please add a comment when you update the code

Reply:
the reference is fine, please also refer to Table 1 and the fact that the keys are 100*C. otherwise it is hard to follow

Reply:
Rather than round to an element of the dictionary, we round as suggested in the reference.